src/java/org/jivesoftware/openfire/muc/spi/MUCPersistenceManager.java

```diff
@@ -129,8 +129,8 @@
     private static final String DELETE_USER_MUCAFFILIATION =
         "DELETE FROM ofMucAffiliation WHERE jid=?";
     private static final String ADD_CONVERSATION_LOG =
-        "INSERT INTO ofMucConversationLog (roomID,sender,nickname,logTime,subject,body,stanza) " +
-        "VALUES (?,?,?,?,?,?,?)";
+        "INSERT INTO ofMucConversationLog (roomID,messageID,sender,nickname,logTime,subject,body,stanza) " +
+        "SELECT ?,COUNT(*),?,?,?,?,?,? FROM ofMucConversationLog";

```

Comment:
This SQL doesn't look right to me, the `COUNT(*)` needs to be in an aggregate function, right?